lib/StripeResource.js

```diff
@@ -40,6 +40,9 @@ StripeResource.prototype = {
 
   path: '',
 
+  // Methods that don't use the API's default '/v1' path can override it with this setting.
+  basePath: null,
```

Comment:
We default to the API's `/v1` prefix unless it's overridden when `extend`-ing `StripeResource`.